analysis/plotting.py

```diff
@@ -184,21 +186,24 @@ def coverage_growth_plot(self,
         axes.set(xlabel='Time (hour:minute)')
 
         if self._logscale or logscale:
```

Comment:
could you better explain the graph changes?

Reply:
For the logscale graphs the chages are to:
1. switch to 'symlog' -- i.e. linear scaling around zero to slightly reduce distortion, seemed slightly better than the default 'log' behavior
2. set the xtick range to start at zero rather than the first snapshot time
3. manually insert a tick for zero itself
4. clear the ticks set by the lineplot default plot

For linear graphs:
1. again set the minimum xtick range to zero
2. ensure that there is always at least one tick (as snapshot time can now be zero)

Both:
ensure the y-axis is at zero

The logscale plots were the best I could come up with playing around with the parameters, but there is an inherent issue with starting them at zero as `lim x-->0 log(x) = -inf`. The main visible result is that the space between 0 and the first snapshot will take up a large portion of the graph. You can see from the logscale graph attached to the PR. I'm open to suggestions to change this, or we can just revert to the old behavior and drop data from t=0 to get the same logscale graphs as before.